spec/reek/smells/too_many_constants_spec.rb

```diff
@@ -0,0 +1,129 @@
+require_relative '../../spec_helper'
+require_lib 'reek/smells/too_many_constants'
+require_relative 'smell_detector_shared'
+
+RSpec.describe Reek::Smells::TooManyConstants do
+  context 'counting constants' do
+    it 'should not report for non-excessive constants' do
+      src = <<-EOS
+        class Dummy
+          A = B = C = D = E = 1
+        end
+      EOS
+
+      expect(src).not_to reek_of(:TooManyConstants)
+    end
+
+    it 'should count each constant only once' do
+      src = <<-EOS
```

Comment:
If you put these three classes together in a module, my bet is the module will smell. I don't think this is what we want.